iterators.tex

```diff
@@ -3723,38 +3717,38 @@
   class move_sentinel {
   public:
     constexpr move_sentinel();
-    explicit move_sentinel(S s);
-    move_sentinel(const move_sentinel<ConvertibleTo<S>>& s);
-    move_sentinel& operator=(const move_sentinel<ConvertibleTo<S>>& s);
+    explicit constexpr move_sentinel(S s);
+    constexpr move_sentinel(const move_sentinel<ConvertibleTo<S>>& s);
+    constexpr move_sentinel& operator=(const move_sentinel<ConvertibleTo<S>>& s);
 
-    S base() const;
+    constexpr S base() const;
```

Comment:
I also don't see these changes to `move_sentinel` in p0579. It's a screwup for sure, but we really should file a defect for this.